region/maridia/outer/Fish Tank.json

```diff
@@ -1990,9 +2006,97 @@
       "link": [7, 6],
       "name": "Base",
       "requires": [
-        "h_canNavigateUnderwater"
+        "Gravity",
+        {"or": [
+          "HiJump",
+          "canSpringBallJumpMidAir",
+          "canDodgeWhileShooting",
+          "canGravityJump",
+          "canCrouchJump",
+          "canDownGrab",
+          {"enemyDamage": {
+            "enemy": "Pink Space Pirate (standing)",
+            "type": "contact",
+            "hits": 1
+          }},
+          {"enemyKill": {
+            "enemies": [["Pink Space Pirate (standing)"]],
+            "explicitWeapons": ["Plasma"]
+          }}
+        ]}
+      ]
+    },
+    {
+      "link": [7, 6],
+      "name": "Suitless Jump Assist",
+      "requires": [
+        "canSuitlessMaridia",
+        {"or": [
+          {"and": [
+            "HiJump",
+            "canSpringBallJumpMidAir"
+          ]},
+          {"and": [
+            "HiJump",
+            "canDodgeWhileShooting"
+          ]},
+          {"and": [
+            "canTrickyJump",
+            "canSpringBallJumpMidAir",
+            {"or": [
+              "canCrouchJump",
+              "canDownGrab"
+            ]}
+          ]}
+        ]}
+      ],
+      "note": [
+        "It is possible to land on the platform with the pirate when it is facing away and shaking its head.",
+        "Then quickly jump away or continuously shoot the pirate to prevent it from shooting."
       ]
     },
+    {
+      "link": [7, 6],
+      "name": "Suitless",
+      "requires": [
+        "canSuitlessMaridia",
+        {"or": [
+          {"enemyKill": {
+            "enemies": [["Pink Space Pirate (standing)"]],
+            "explicitWeapons": ["Plasma"]
+          }},
+          {"and": [
+            "canUseIFrames",
+            {"enemyDamage": {
+              "enemy": "Pink Space Pirate (standing)",
+              "type": "contact",
+              "hits": 1
+            }}
+          ]}
+        ]},
+        {"or": [
+          "HiJump",
+          "canSpringBallJumpMidAir",
+          "h_canCrouchJumpDownGrab",
+          {"and": [
+            "Plasma",
+            "canTrickyUseFrozenEnemies"
```

Comment:
canUseFrozenEnemies

Reply:
I was on the fence about this since it's a bit non-intuitive, and if you kill the pirate you likely cant reset the room. But I think its fine to lower it. I also split the tank/kill strats, it looked like tank+spring was not great.